hasher-matcher-actioner/src/OpenMediaMatch/blueprints/hashing.py

```diff
@@ -124,9 +124,18 @@ def hash_media() -> dict[str, str]:
 
     try:
         # Get response with content length tracking
-        max_content_length = int(
-            current_app.config.get("MAX_CONTENT_LENGTH", DEFAULT_MAX_CONTENT_LENGTH)
-        )
+        max_content_length = current_app.config.get("MAX_CONTENT_LENGTH")
```

Comment:
only other option would be to change the setting const - that would simplify a lot of this since we'd fall back to the `get` 

that might be a breaking change for anyone who is using it, though :(

Reply:
Or is `MAX_CONTENT_LENGTH` used by Flask somewhere, and that's why it's defaulting to None?

Reply:
yeah, MAX_CONTENT_LENGTH is already a config used by Flask - https://flask.palletsprojects.com/en/stable/config/#MAX_CONTENT_LENGTH

if we use something else, like MAX_CONTENT_LENGTH_HMA, the `.get` default would work as expected

Reply:
Okay, I'll update to reflect that. Wasn't sure if you'd chosen `MAX_CONTENT_LENGTH` to give parity with the POST multipart endpoint or something.

Will be a "breaking" change, but should be fine.